search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java

```diff
@@ -411,6 +417,81 @@ void shouldReturnResourcesForSinglePeriods() throws BadRequestException {
+            var firstLevelOfViewingScope = URLEncoder.encode("https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.0.0",
+                                                 StandardCharsets.UTF_8);
+            var secondLevelOfViewingScope = URLEncoder.encode("https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.1.0",
+                                                 StandardCharsets.UTF_8);
+            var query =
+                ResourceQuery.builder()
+                    .fromQueryParameters(Map.of(VIEWING_SCOPE.fieldName(), firstLevelOfViewingScope
+                                                                           + "," + secondLevelOfViewingScope,
+                                                EXCLUDE_SUBUNITS, Boolean.TRUE.toString()))
+                    .withRequiredParameters(FROM, SIZE, AGGREGATION)
+                    .withOpensearchUri(URI.create(container.getHttpHostAddress()))
+                    .build()
+                    .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA, DELETED);
+
+            logger.info(query.getValue(SORT).toString());
+            var response = searchClient.doSearch(query);
+            var pagedSearchResourceDto = query.toPagedResponse(response);
+
+            var excludedSubunit = "https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.1.1";
+
+            assertThat(pagedSearchResourceDto.toJsonString(), not(containsString(excludedSubunit)));
+            assertThat(pagedSearchResourceDto.hits(), hasSize(2));
+        }
+
+        @Test
+        void shouldReturnResourcesWithSubunitsWhenExcludedSubunitsSearchParamIsNotProvided() throws BadRequestException {
+            var viewingScope = URLEncoder.encode("https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.0.0",
+                                                 StandardCharsets.UTF_8);
+            var query =
+                ResourceQuery.builder()
+                    .fromQueryParameters(Map.of(VIEWING_SCOPE.fieldName(), viewingScope))
+                    .withRequiredParameters(FROM, SIZE, AGGREGATION)
+                    .withOpensearchUri(URI.create(container.getHttpHostAddress()))
+                    .build()
+                    .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA, DELETED);
+
+            logger.info(query.getValue(SORT).toString());
+            var response = searchClient.doSearch(query);
+            var pagedSearchResourceDto = query.toPagedResponse(response);
+
+            var includedSubunitI = "https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.1.0";
+            var includedSubunitII = "https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.1.1";
+
+            assertThat(pagedSearchResourceDto.toJsonString(), containsString(includedSubunitI));
+            assertThat(pagedSearchResourceDto.toJsonString(), containsString(includedSubunitII));
+            assertThat(pagedSearchResourceDto.hits(), hasSize(3));
+        }
```

Comment:
I find it strange that these docker-tests pass without you needing to alter test_resources_mappings.json

Reply:
If the content is "structurally congruent" and no other data is uploaded, it likely makes sense